3rdparty/libgcov/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+add_enclave_library(gcov STATIC
+  # Only the following three files are required to support GCOV-based code coverage analysis.
+  src/GCDAProfiling.c
+  src/InstrProfiling.c
+  src/InstrProfilingUtil.c)
+
+maybe_build_using_clangw(gcov)
+
+enclave_compile_options(gcov PRIVATE -Wno-error)
```

Comment:
Would you have to say do not use system headers?

Reply:
Are you referring to the potential conflicts?